tat2xmpp/api/xmpp.go

```diff
@@ -19,31 +19,44 @@ func serverName(host string) string {
 }
 
 func getNewXMPPClient() (*xmpp.Client, error) {
-	xmpp.DefaultConfig = tls.Config{
-		ServerName:         serverName(viper.GetString("xmpp_server")),
-		InsecureSkipVerify: viper.GetBool("xmpp_insecure_skip_verify"),
-	}
+	hosts := strings.Split(viper.GetString("xmpp_server"), ",")
 
-	options := xmpp.Options{Host: viper.GetString("xmpp_server"),
-		User:          viper.GetString("xmpp_bot_jid"),
-		Password:      viper.GetString("xmpp_bot_password"),
-		NoTLS:         viper.GetBool("xmpp_notls"),
-		StartTLS:      viper.GetBool("xmpp_starttls"),
-		Debug:         viper.GetBool("xmpp_debug"),
-		Session:       viper.GetBool("xmpp_session"),
-		Status:        "",
-		StatusMessage: "",
-	}
+	var xmppClient *xmpp.Client
+	for index, host := range hosts {
+		xmppClient = nil
+		xmpp.DefaultConfig = tls.Config{
+			ServerName:         serverName(host),
+			InsecureSkipVerify: viper.GetBool("xmpp_insecure_skip_verify"),
+		}
 
-	xmppClient, err := options.NewClient()
+		options := xmpp.Options{
+			Host:          host,
+			User:          viper.GetString("xmpp_bot_jid"),
+			Password:      viper.GetString("xmpp_bot_password"),
+			NoTLS:         viper.GetBool("xmpp_notls"),
+			StartTLS:      viper.GetBool("xmpp_starttls"),
+			Debug:         viper.GetBool("xmpp_debug"),
+			Session:       viper.GetBool("xmpp_session"),
+			Status:        "",
+			StatusMessage: "",
+		}
 
-	if err != nil {
-		log.Panicf("getClient >> NewClient XMPP, err:%s", err)
-		return nil, err
+		var errNewClient error
```

Comment:
I need to initialize this variable because I'm using `xmppClient, errNewClient = options.NewClient()` to affect a value to xmppClient which is initialized outside of the loop